### ISSUE TITLE
fs-promise warns that it is deprecated 

### ISSUE THREAD

**Issue by BrandonOCasey** [marcoscaceres/karma-safaritechpreview-launcher#8]
The following message is displayed upon installing this dependency: `npm WARN deprecated fs-promise@0.5.0: Use mz or fs-extra^3.0 with Promise Support`

Would a pull request switching to `fs-promise` to `fs-extra` or `mz` be accepted?

**Comment by marcoscaceres** [comment]
@BrandonOCasey, absolutely! Alternatively, you could `utils.promisify()` whatever `fs-promise` is currently doing. Then we could drop that dependency entirely. Let me know!

**Comment by BrandonOCasey** [comment]
@marcoscaceres I agree that `util.promisify` would be better. That would probably require another major version, since you would be dropping node support <8. Is that OK?

**Comment by marcoscaceres** [comment]
yep, that's totally fine.
